pkg/services/ngalert/state/manager.go

```diff
@@ -239,7 +273,7 @@ func (st *Manager) setNextState(ctx context.Context, alertRule *ngModels.AlertRu
 		logger.Debug("Ignoring set next state as result is pending")
 	}
 
-	// Set reason iff: result is different than state, reason is not Alerting or Normal
+	// Set reason iff: result and state are different, reason is not Alerting or Normal
```

Comment:
```suggestion
	// Set reason if result and state are different, reason is not Alerting or Normal
```

Reply:
`iff` in the abbreviation of `if and only if`

Reply:
I was not sure whether it was a typo or an "if and only if" 😁